xum1541/Makefile

```diff
@@ -83,10 +83,14 @@ MODELVERSION= xum1541-$(MODEL)-v$(XUMFW_VERSION)
 # Resulting full revision number for model and firmware version
 REVISION=$(shell grep '\#define $(MODEL)' xum1541.h | sed 's/\#define $(MODEL)//' | xargs printf "%02X")$(XUMFW_VERSION)
 
+# git revision or "unknown" if either not a git repo or git command unavailable
+GITREV=$(shell git describe --always --exclude '*' || echo 'unknown')
```

Comment:
Here, there are two additional cases that should be handled:
1. git is not installed at all
2. git is installed, but the compilation is not done from a git repository

Reply:
I'm not sure I understand, because I think they **are** handled? As the comment says, both cases should result in `GITREV` being set to the string "unknown". This works for me. Doesn't it work for you? Or do you think we could do better?

Reply:
*argh*
You're right, I must have been blind yesterday.